model/document.js

```diff
@@ -129,6 +129,38 @@ module.exports = memoize(function (db) {
 		isReleased: { type: db.Boolean, value: function () {
 			return this.status === 'approved';
 		} },
+		toWebServiceJSON: {
+			value: function (ignore) {
+				var data = {
+					code: this.key,
```

Comment:
this is "document" in uploads and payments cases:
https://gist.github.com/roxanadina/d8b5c7619f84fedea835386a7d2511f2#file-new-bp-json-L76